frontend/src/components/SignIn/SignIn.js

```diff
@@ -12,6 +12,9 @@ class SignIn extends Component {
       isPasswordVisible: false,
       email: '',
       password: '',
+      errorText: 'Please Vish is a programmer!',
```

Comment:
It looks like `errorText` isn't being used anywhere, so I think we can delete this line.